app/Http/Controllers/Admin/AdminController.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Remove all files from public/admin which are not related to implemented theme. I guess css, js, scss, assets, images are the only required one. 
.mjs, .astro, .mdx, I am not aware if such files being used in theme UI anyhow.